soc_039_rw1_out_of_school_rate/soc_039_rw1_out_of_school_rate_processing.py

```diff
@@ -0,0 +1,117 @@
+import pandas as pd
+import os
+import urllib.request
+import sys
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+import util_files
+import util_cloud
+import util_carto
+from zipfile import ZipFile
+import logging
+import datetime
+import glob
+import shutil
+  
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
+# this should be a table name that is not currently in use
+dataset_name = 'soc_039_rw1_out_of_school_rate'
+
+logger.info('Executing script for dataset: ' + dataset_name)
+# create a new sub-directory within your specified dir called 'data'
+# within this directory, create files to store raw and processed data
+data_dir = util_files.prep_dirs(dataset_name)
+
+'''
+Download data and save to your data directory
+
+Data can be downloaded at the following link:
+http://data.uis.unesco.org/Index.aspx
+From the left panel, click to select 'EDUCATION' - 'Sustainable Development Goals 1 and 4' - '4.1.4 Out-of-school rate'.
```

Comment:
From the left panel, click to select 'EDUCATION' - 'Sustainable Development Goals 1 and 4' - 'Sustainable Development Goal 4' - 'Target 4.1' - '4.1.4 Out-of-school rate' - '4.1.4 Out-of-school rate by school age and sex (administrative data)'